index.js

```diff
@@ -11,22 +11,25 @@ exports.viewcounter = async (request, response) => {
   response.set("Access-Control-Allow-Origin", "*");
   response.set("Access-Control-Allow-Methods", "GET");
 
-  const referrer = request.get("referrer");
+  const href = request.body;
 
-  if (!referrer) {
-    response.status(BAD_REQUEST).json({
-      message: "referrer is undefined!",
+  return response.json({ blah: 'blah' });
```

Comment:
**question**: what is going on here?

Reply:
Oh man. That's embarrassing. It was a last second test to make sure that I could remove the watch stuff. 😬